### DISCUSSION TITLE
Bots

### DISCUSSION THREAD

**Discussion by Chromaah** [diamond-rip/Eden#354] [Q&A] [answered]
Is it possible to create bots with the eden API? @GoodestEnglish 

If so, anyone experienced willing to code an addon? (would use the spigots kb for the bot, my case carbonspigot) 

Would pay.

**Comment by GoodestEnglish** [comment, marked as answer]
Yes it is possible. I have tried to code a very simple (fightable at least) boxing/nodebuff bot before but it is discontinued. If you have a developer and really want to, I can sell mine w/ SRC for $10 paypal. The bot works with citizens and do basic fight/pot. Dm me on discord if you are interested.

Edit: The plugin + source is not resell-able, and will not be maintained by me anymore.

**Reply by Chromaah** [reply]
I dmed you. and i wont re-sell
